kafka-sample-temperature-producer/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/sumudul
```

Comment:
Shall we use a module name that matches the repo url?
e.g: https://github.com/wso2/choreo-samples/blob/main/go-reading-list-rest-api/go.mod#L1